content/6_Advanced/Multiplicative.mdx

```diff
@@ -64,6 +267,335 @@ int main() {
 }
 ```
 
-(project euler)
+</CPPSection>
+</LanguageSection>
+
+## Example - Enumerate Primes
```

Comment:
what does this example have to do with prefix sums of multiplicative functions? can this be removed?

Reply:
okay :D orz (it was in the foucs problem thing that was there before I touched the page so upon request I just completed it)